stan/math/prim/mat/fun/exp2.hpp

```diff
@@ -0,0 +1,46 @@
+#ifndef STAN_MATH_PRIM_MAT_FUN_EXP2_HPP
+#define STAN_MATH_PRIM_MAT_FUN_EXP2_HPP
+
+#include <stan/math/prim/mat/vectorize/apply_scalar_unary.hpp>
+#include <stan/math/prim/scal/fun/exp2.hpp>
+
+namespace stan {
+  namespace math {
+
+    /**
+     * Structure to wrap exp2() so it can be vectorized.
+     */
+    struct exp2_fun {
+      /**
+       * Return the inverse hypberbolic cosine of the specified argument.
```

Comment:
Just happened to notice that the comment here looks like it's referring to `acosh` rather than `exp2`.